src/ducks/apps/components/UninstallModal.jsx

```diff
@@ -18,24 +18,25 @@ export class UninstallModal extends Component {
 
   uninstallApp () {
     this.setState({ error: null })
-    const { app } = this.props
+    const { app, history } = this.props
     this.props.uninstallApp(app.slug)
     .then(() => {
-      this.gotoParent()
+      history.push('/myapps')
```

Comment:
Here again that should go to the parent, we can uninstall an installed app from the discover view as well.